tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/wwwroot/app/dashboard/settings.controller.js

```diff
@@ -367,6 +377,7 @@
                                     vm.activeDocumentTypeLawTerm = levelTwoTerm;
                                 }
                                 vm.selectedDocumentTypeLawTerms.push(documentType);
+                                getAdditionalMatterProperties(documentType);

```

Comment:
Pass the term or Area of law to get the Additional matter fields associated to it.